tests/plugins/uploadwidget/uploadwidget.js

```diff
@@ -156,6 +156,26 @@
 			} );
 		},
 
+		'test destroy editor during upload': function() {
+
+			bender.editorBot.create( { name: 'destroy_test', config: { extraPlugins: 'uploadwidget' } }, function( bot ) {
+				var editor = bot.editor;
+
+				addTestUploadWidget( editor, 'testuploadwidget' );
+				pasteFiles( editor, [ bender.tools.getTestPngFile() ] );
+
+				var loader = editor.uploadRepository.loaders[ 0 ];
+
+				try {
+					editor.destroy();
```

Comment:
I'm not sure if it's really "destroying while uploading". In reality dropping image generates three states:

1. `loading`
2. `loaded`
3. `uploading`

Destroying editor via button in manual test is in fact possible only **after** switching to `uploading`. What's more, error is thrown not while changing state, but while handling `update` event → https://github.com/ckeditor/ckeditor-dev/blob/f4e106d5736e5d4211f7371a863d17dd46270e30/plugins/uploadwidget/plugin.js#L257